MarkSkalskyi/README.md

```diff
@@ -0,0 +1,42 @@
+# Getting started with [HDRezka](https://rezka.ag)
+
+HDRezka is a website that allows you to watch your favorite movies and series. If you need a guide for signing in, you're in the right place.
+
+**Movies? HDRezka!**
```

Comment:
This text doesn't provide any useful information in the README. I suggest removing it.